### ISSUE TITLE
In Scan mode, Screen reader is not announcing the ItemName/tagname associated with 'remove' button in the tag picker or people picker..

### ISSUE THREAD

**Issue by azra-bandukwala** [microsoft/fluentui#15903]
User Experience:

Users who rely on screen readers are impacted and face difficulty if screen reader does not announces the current dialog information.

DT Test Environment:
OS: Windows 10
Version: 2004(OS Build 20206.1000)
Browser: Edge Dev 
Browser Version: 87.0.634.0 (Official build) dev (64-bit)
Screen Reader: Narrator
URL: https://portal-sdf.office.com/AdminPortal/Home#/featureexplorer

Repro Steps:

Step 1: Open above URL in Edge Dev browser and log in with user account: admin@a830edad9050849829J20060916.onmicrosoft.com

Step 2: Navigate to 'view' button in connect people to knowledge and activate it.

Step 3: In 'Connect people to knowledge' page navigate to 'Manage' button and activate it.

Step 4:  In knowledge network window navigate to 'Topic discovery' tab item.

Step 5:  Navigate to 'Edit' button of 'Select SharePoint topic sources' and activate it with enter key and observe the issue.

Actual:
Screen reader remains silent while activating 'Edit' button of 'Select SharePoint topic sources' button.

Note: Issue is repro with all the tabs of Knowledge network dialog.

Expected:
Screen reader should announce the dialogue content when the user activates the button, so that the user can get to know that the dialog is opened.

MAS Reference

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Parent bug link - https://office.visualstudio.com/OC/_workitems/edit/4618658

**Comment by paulgildea** [comment]
@azra-bandukwala Thanks for filing. Could you please provide which version of the library you are using or provide a reduced isolated repo?

@behowell Please determine if there is a repo with the website example - this should help us understand if it's in the latest  version.

**Comment by azra-bandukwala** [comment]
@paulgildea -"office-ui-fabric-react": "7.135.0"
